src/toil/test/wdl/wdltoil_test.py

```diff
@@ -55,6 +56,29 @@ def tearDown(self) -> None:
     64, # Legacy test for as_map_as_input; It looks like MiniWDL does not have the function as_map()
     77, # Test that array cannot coerce to a string. WDL 1.1 does not allow compound types to coerce into a string. This should return a TypeError.
 ]
+WDL_UNIT_TESTS_UNSUPPORTED_BY_TOIL = [
+    14,  # test_object, Objects are not supported
+    19,  # map_to_struct, miniwdl cannot coerce map to struct, https://github.com/chanzuckerberg/miniwdl/issues/712
+    52,  # relative_and_absolute, needs root to run
+    58,  # test_gpu, needs gpu to run, else warning
+    59,  # will be fixed in #5001
+    66,  # This needs way too many resources (and actually doesn't work?), see https://github.com/DataBiosphere/wdl-conformance-tests/blob/2d617b703a33791f75f30a9db43c3740a499cd89/README_UNIT.md?plain=1#L8
+    67,  # same as above
+    68,  # Bug
```

Comment:
Which bug is this? How will we be able to tell when it is fixed?

Reply:
This should be related to https://github.com/DataBiosphere/toil/issues/4993